src/komponenter/meldekort/meldekort.tsx

```diff
@@ -1,20 +1,22 @@
 import React  from 'react';
 import LenkepanelMedIkon from '../lenkepanel-med-ikon/lenkepanel-med-ikon';
-import { gaTilMeldekort } from '../../metrics/metrics';
+import { gaTilMeldekort, loggAktivitet } from '../../metrics/metrics';
 import EmailText from './email-text';
 import { meldekortLenke } from '../../innhold/lenker';
 import { OppfolgingContext } from '../../ducks/oppfolging';
 import { FeaturetoggleContext } from '../../ducks/feature-toggles'
 
-const Meldekort = () => {
+const Meldekort = (props: any) => {
```

Comment:
`any` her og.

Reply:
til ære for deg skal jeg faktisk lage disse 2 interfacene